.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pyc
 .DS_Store
 _site
+.idea/
```

Comment:
Good tip- thanks. Reverted the change to gitignore accordingly.

DS_Store is also a very platform specific entry- should I remove that one while I'm at it?